scripts/mail-alarm

```diff
@@ -11,18 +11,18 @@
 # the only thing that needs be set is pool:other-config:ssmtp-mailhub
 
 from __future__ import print_function
-import XenAPI
-import sys
+
+import json
 import os
+import re
+import sys
+import syslog
 import tempfile
 import traceback
-import syslog
-import json
-import re
-from xml.dom import minidom
-from xml.sax.saxutils import unescape
-from xml.parsers.expat import ExpatError
 from socket import getfqdn
+
+import XenAPI
+from xml.dom.minidom import parseString
```

Comment:
The Import rule from PEP 8 at https://peps.python.org/pep-0008/#imports says:

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

Accordingly, `isort` sorts `from xml.dom.minidom import parseString` directly after `from socket import getfqdn` because it is a 3rd party import, not a local application/library specific import like `import XenAPI` and `from xcp import branding`:
```suggestion
from xml.dom.minidom import parseString

import XenAPI
```